docker/README.md

```diff
@@ -0,0 +1,44 @@
+This document is written for people who are eager to do something with 
+the Exchange Union Daemon (`xud`). This folder uses `docker-compose` to
+package `xud`, `mysql`, `lnd` & `btcd` together to make deploying the two services as easy as
+typing a few commands. All configuration between `xud`, `mysql` and other containers are handled
+automatically by their `docker-compose` config file.
+
+The goal of this is that `docker-compose` to be able to spin up / lauch `xud` along with `lnd` & `Raiden` networks.
```

Comment:
instead of "along with `lnd` & `Raiden` networks", perhaps specify that it's for launching `xud` along with all its dependencies.